### PR TITLE
Skeleton code structure for Glue L2 construct

### PR DESCRIPTION
**Pull request by mjanardhan**
> REPLACE THIS TEXT BLOCK
>
> Describe the reason for this change, what the solution is, and any
> important design decisions you made. 
>
> Remember to follow the [CONTRIBUTING GUIDE] and [DESIGN GUIDELINES] for any
> code you submit.
>
> [CONTRIBUTING GUIDE]: https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md
> [DESIGN GUIDELINES]: https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md

Closes #<issue number here>.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-glue-alpha/lib/conditionaltrigger.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/conditionaltrigger.ts:1]: Rename to conditional-trigger.ts - best practice to separate words by hyphens for readability.

File: packages/@aws-cdk/aws-glue-alpha/lib/flexjob.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/flexjob.ts:1]: flex-job.ts

Comment (natalie-white-aws) [reply packages/@aws-cdk/aws-glue-alpha/lib/flexjob.ts:1]: Create a sub-folder "jobs" to hold all of the Job related classes.

File: packages/@aws-cdk/aws-glue-alpha/lib/conditionaltrigger.ts
Comment (natalie-white-aws) [reply packages/@aws-cdk/aws-glue-alpha/lib/conditionaltrigger.ts:1]: Create sub-folder for trigger files

File: packages/@aws-cdk/aws-glue-alpha/lib/notifyeventtrigger.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/notifyeventtrigger.ts:1]: notify-event-trigger.ts

File: packages/@aws-cdk/aws-glue-alpha/lib/ondemandtrigger.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/ondemandtrigger.ts:1]: on-demand-trigger.ts

File: packages/@aws-cdk/aws-glue-alpha/lib/pythonshelljob.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/pythonshelljob.ts:1]: python-shell-job.ts

File: packages/@aws-cdk/aws-glue-alpha/lib/rayjob.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/rayjob.ts:1]: ray-job.ts

File: packages/@aws-cdk/aws-glue-alpha/lib/scheduledtrigger.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/scheduledtrigger.ts:1]: scheduled-trigger.ts

File: packages/@aws-cdk/aws-glue-alpha/lib/sparketljob.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/sparketljob.ts:1]: spark-etl-job.ts

File: packages/@aws-cdk/aws-glue-alpha/lib/streamingjob.ts
Comment (natalie-white-aws) [inline packages/@aws-cdk/aws-glue-alpha/lib/streamingjob.ts:1]: streaming-job.ts

**Review by natalie-white-aws — CHANGES_REQUESTED** [review]
Create trigger and job sub-directories to separate classes; add hyphens to file names for readability.

File: packages/@aws-cdk/aws-glue-alpha/lib/conditionaltrigger.ts
Comment (mjanardhan) [reply packages/@aws-cdk/aws-glue-alpha/lib/conditionaltrigger.ts:1]: done

**Review by natalie-white-aws — APPROVED** [review]
Approved.

**Pull request merged**